device_backends/NumericAddressedBackend/src/NumericAddressedBackend.cc

```diff
@@ -3,15 +3,15 @@
 
 #include "NumericAddressedBackend.h"
 
-#include "AsyncDomainImpl.h"
-#include "AsyncDomainsContainer.h"
+#include "async/DomainImpl.h"
+#include "async/DomainsContainer.h"
+#include "async/SubDomain.h"
 #include "Exception.h"
 #include "MapFileParser.h"
 #include "NumericAddress.h"
 #include "NumericAddressedBackendASCIIAccessor.h"
 #include "NumericAddressedBackendMuxedRegisterAccessor.h"
 #include "NumericAddressedBackendRegisterAccessor.h"
-#include "TriggerDistributor.h"
 #include <nlohmann/json.hpp>
```

Comment:
The json include is no longer required here, I guess?

Reply:
removed